assets/css/_leaflet.scss

```diff
@@ -1,5 +1,5 @@
-// From here: https://github.com/Leaflet/Leaflet/blob/5c29d615563835d373a51b91fc3513a17907da96/dist/leaflet.css#L87
-$leaflet-pane-z-index: 400;
+// From here: https://github.com/Leaflet/Leaflet/blob/5c29d615563835d373a51b91fc3513a17907da96/dist/leaflet.css
```

Comment:
question(non-blocking): just curious why you removed the line info? is it worth linking to the new line which has the z-index value we're referencing here?

Reply:
For some reason I thought there were a couple of places in the file with `z-index: 1000`, but it looks like there's just one, so I'll link to it!